modules/statistics/php/widgets.class.inc

```diff
@@ -65,14 +65,25 @@ class Widgets extends \NDB_Page implements ETagCalculator
 
         $recruitmentTarget = $config->getSetting('recruitmentTarget');
 
-        $totalScans  = $this->_getTotalRecruitment($db);
-        $recruitment = [
+        $recruitmentRaw = $db->pselect(
+            "SELECT
+                COUNT(*) as Count,
+                c.Sex as Sex,
+                c.RegistrationProjectID as ProjectID
+             FROM candidate c
+                 WHERE c.Active='Y' AND c.Entity_type='Human'
+                 AND c.RegistrationCenterID <> 1
+             GROUP BY c.Sex, c.RegistrationProjectID",
+            []
+        );
+        $totalScans     = $this->_getTotalRecruitment($recruitmentRaw);
```

Comment:
Can this just be done with array_count / array_column instead of having the additional function? This way we could remove code clutter by removing that function.
```suggestion
        $totalScans     = array_count(array_column($recruitmentRaw, 'Count'));
```

Reply:
I assumed you meant array_sum, not array_count, and made this change.

Reply:
Yes sorry! I think i got "array_count" in my head because the column name was "count". thanks!